requirements.txt

```diff
@@ -0,0 +1 @@
+mkdocs
```

Comment:
I would remove this file entirely. This will cause confusion if it is in the devel branch. I have changed the pip command in the github action to directly install mkdocs rather than running pip -r requirement.txt.